lib/matplotlib/sankey.py

```diff
@@ -168,15 +169,14 @@ def _arc(self, quadrant=0, cw=True, radius=1, center=(0, 0)):
         Return the codes and vertices for a rotated, scaled, and translated
         90 degree arc.
 
-        Optional keyword arguments:
-
-          ===============   ==========================================
-          Keyword           Description
-          ===============   ==========================================
-          *quadrant*        uses 0-based indexing (0, 1, 2, or 3)
-          *cw*              if True, clockwise
-          *center*          (x, y) tuple of the arc's center
-          ===============   ==========================================
+        Other Parameters
+        ----------------
+        quadrant : int, {0, 1, 2, 3}
```

Comment:
I guess the only acceptable values are 0, 1, 2, 3, not any int.
(Also perhaps state the defaults.)

Reply:
I'm not really sure what 0, 1, 2, 3 mean, and I'm getting the feeling the optimization is not really worth it and we should just replace this function with `Path.arc`... But that would be a separate PR.